client.go

```diff
@@ -248,6 +248,14 @@ type Client interface {
 
 	// GetSchema() returns ovn-db schema
 	GetSchema() libovsdb.DatabaseSchema
+
+	// AuxKeyValSet() sets keys/values for a column of OvsMap type, e.g., 'external_ids', 'other_config'.
+	AuxKeyValSet(table string, row string, auxCol string, kv map[string]string) (*OvnCommand, error)
```

Comment:
For the argument "row", in this PR implementation it is expecting "name" (instead of UUID), so maybe it is better to make it clear in the argument naming, e.g. "rowName".

(It is also possible to use UUID, and the "name" v.s. "UUID" discussion is still on going for other APIs.)

Reply:
ack. will fix.